Directory.Build.props

```diff
@@ -21,6 +21,10 @@
     <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(RootIntDir)\$(Platform)\$(Configuration)</BaseIntDir>
     <BaseOutDir Condition="'$(Platform)' == 'Win32'">$(RootOutDir)\x86\$(Configuration)</BaseOutDir>
     <BaseOutDir Condition="'$(Platform)' != 'Win32'">$(RootOutDir)\$(Platform)\$(Configuration)</BaseOutDir>
+
+    <ReactNativeRootIntDir>$(ReactNativeWindowsDir)build\</ReactNativeRootIntDir>
+    <ReactNativeRootOutDir>$(ReactNativeWindowsDir)target\</ReactNativeRootOutDir>   
```

Comment:
There is already a `RootOutDir` defined above (which is often overwritten by CI).

```suggestion
```

Reply:
Good point! I can address the issue with prefixes in future PRs. Remove the duplicates for now.